ext/fetch/23_request.js

```diff
@@ -554,7 +559,7 @@ function toInnerRequest(request) {
  * @returns {Request}
  */
 function fromInnerRequest(inner, signal, guard) {
-  const request = webidl.createBranded(Request);
+  const request = new Request(webidl.brand);
   request[_request] = inner;
   request[_signal] = signal;
   request[_getHeaders] = () => headersFromHeaderList(inner.headerList, guard);
```

Comment:
Because of the way v8 optimizes classes, it _might_ be faster to pass all of these into the constructor after the brand, so that the object always has the same shape.

Reply:
I'll run some benchmarks